python/xorbits/core/data.py

```diff
@@ -40,6 +41,12 @@ class DataType(Enum):
 DATA_MEMBERS: Dict[DataType, Dict[str, Any]] = defaultdict(dict)
 
 
+class ConversionType(Enum):
+    int_conversion = 1
+    float_conversion = 2
+    bool_conversion = 3
+
```

Comment:
Define a method to convert value would be more clean.
``` Python
class ConversionType(Enum):
    int_conversion = 1
    float_conversion = 2
    bool_conversion = 3

    def convert(self, val):
        if self.value == 1:
            return int(val)
        ...
```
For line273 to line 278, just call `conversion.convert(self.to_numpy())`.